.github/workflows/tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "pypy3"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
Why remove pypy?

Reply:
The previous tests claimed it was 3.6, i.e., it no longer exists.